packages/libs/hd-wallet/src/index.ts

```diff
@@ -1 +1,253 @@
 export * from './utils';
+import type { IUnsignedCommand } from '@kadena/client';
+import { verifySig } from '@kadena/cryptography-utils';
+import * as bip39 from '@scure/bip39';
+import { wordlist } from '@scure/bip39/wordlists/english';
+import {
+  decryptPrivateKey,
+  encryptOrEncodeSeed,
+  encryptPrivateKey,
+  extractSeedBuffer,
+} from './utils/encrypt';
+import { deriveKeyPair, signWithKeyPair, signWithSeed } from './utils/sign';
+
+/**
+ * Changes the password of an encrypted private key.
+ *
+ * @param {string} privateKey - The encrypted private key as a Base64 encoded string.
+ * @param {string} oldPassword - The current password used to encrypt the private key.
+ * @param {string} newPassword - The new password to encrypt the private key with.
+ * @returns {string} - The newly encrypted private key as a Base64 encoded string.
+ * @throws {Error} - Throws an error if the old password is empty, new password is incorrect empty passwords are empty, or if encryption with the new password fails.
+ */
+export function kadenaChangePassword(
+  privateKey: string,
+  oldPassword: string,
+  newPassword: string,
+): string {
```

Comment:
```suggestion
export function kadenaChangePassword(
  encryptedMessageWithSalt: string,
  oldPassword: string,
  newPassword: string,
): string
```

Using the `kadenaEncrypt` and `kadenaDecrypt` functions